kafka-streams-framework/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    `java-library`
+    jacoco
+    id("com.commercehub.gradle.plugin.avro") version "0.9.1"
+    id("org.hypertrace.publish-plugin")
+    id("org.hypertrace.jacoco-report-plugin")
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+dependencies {
+    api("org.apache.kafka:kafka-streams:5.5.1-ccs")
+    api("com.typesafe:config:1.4.0")
+    api("io.confluent:kafka-avro-serializer:5.5.0")
+    implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.0")
+    implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.5")
+    implementation("org.apache.kafka:kafka-clients:5.5.1-ccs")
+    testImplementation("org.apache.kafka:kafka-streams-test-utils:5.5.1-ccs")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.6.2")
+    testImplementation("org.mockito:mockito-core:3.3.3")
+    testImplementation("org.hamcrest:hamcrest-core:1.3")
```

Comment:
what's hamcrest for?

Reply:
hamcrest is a rich library for building matching logic that could be used in assertions: https://en.wikipedia.org/wiki/Hamcrest.